src/cloud/install/freedom-module.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Cloud Installer",
+  "description": "Installs uProxy in the cloud, via SSH.",
+  "app": {
+    "script": [
+      "freedom-module.static.js"
+    ]
+  },
+  "provides": [ "cloudinstall" ],
+  "default": "cloudinstall",
+  "api": {
+    "cloudinstall": {
+      "install": {
+        "type": "method",
+        "value": ["string", "number", "string", "string"],
+        "ret": "string"
+      },
+      "err": {
+        "message": "string"
+      }
+    }
+  },
+  "permissions": [
+    "core.tcpsocket"
```

Comment:
Is the ssh2 package successfully depending on this? I don't see any explicitly instantiated socket, though clearly the dependency will be needed.